packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -153,17 +157,181 @@ class GtagExceptionDevTools extends GtagException {
   external String get ide_launched;
 }
 
+/// Request Flutter tool stored value enabled (GA enabled) stored in ~\.flutter.
+/// Return bool or null, null implies Flutter Tool has never been run.
```

Comment:
dart doc: add blank line between first sentence and subsequent sentences

Reply:
Done.